src/DeviceRendererFactory.js

```diff
@@ -156,35 +154,25 @@ module.exports = class DeviceRendererFactory {
     }
 
     /**
-     * Loads the selected template.
+     * Loads HTML template.
      *
      * @param  {HTMLElement}        dom       The DOM element to setup the device renderer into.
-     * @param  {string}             template  Template to use.
      * @param  {Object}             options   Various configuration options.
      */
-    loadTemplate(dom, template, options) {
-        const head = document.getElementsByTagName('head')[0];
-        const scriptId = TEMPLATE_JS + '-' + options.template;
-        const styleId = TEMPLATE_CSS + '-' + options.template;
-
-        // Handle template JS
-        if (!document.getElementById(scriptId)) {
-            const script = document.createElement('script');
-            script.id = scriptId;
-            script.text = template.js;
-            head.appendChild(script);
-        }
-
-        // Handle template CSS
-        if (!document.getElementById(styleId)) {
-            const style = document.createElement('style');
-            style.id = styleId;
-            style.appendChild(document.createTextNode(template.css));
-            head.appendChild(style);
-        }
-
-        // Handle template dom
-        dom.innerHTML = template.html;
+    loadTemplate(dom, options) {
```

Comment:
Shadow should only need to override this function I guess?

Reply:
Right, and inject their own HTML. I wonder if it would be more logical to move loadTemplate to DeviceRenderer.js